src/roiextractors/multiimagingextractor.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, imaging_extractors: List[ImagingExtractor]):
         # Checks that properties are consistent between extractors
         self._check_consistency_between_imaging_extractors()
 
+        self._dtype = self._imaging_extractors[0].get_dtype()
```

Comment:
Maybe I'll just overload the function itself instead... WIP

Reply:
Yes, it is a better idea. If we are using this perhaps we should check also for dtype consistency in `_check_consistency_between_imaging_extractors`?

Reply:
Oh good catch, didn't even notice that

Reply:
Added to consistency check and defined override method

Reply:
thanks!